js/inputbar.js

```diff
@@ -294,6 +294,13 @@ weechat.directive('inputBar', function() {
                     return true;
                 }
 
+                if ($event.altKey && (code === 38 || code === 40)) {
```

Comment:
Good catch! The current state interferes with workspace switching for me. Please check that Ctrl isn't pressed a swell.